use std::error::Error;

mod data;

[tokio::main]
async fn main() -> Result<(), Error> {
}
